mimecast/plugin.spec.yaml

```diff
@@ -701,8 +701,8 @@ actions:
         - malicious
         - all
       url_to_filter:
-        title: URL filter
-        description: URL to filter results with
+        title: Filter
```

Comment:
Changing the title of input/outputs is a backward-incompatible change since any workflows will no longer work after a plugin update if they depend on that field.  You'll need to do a major version bump

Reply:
ok I thought if we did not change the actual variable name we would be ok. bummer :(

Reply:
It's because the workflow language uses `{{[Step Title].[Title]}}`